bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -35,7 +35,7 @@ <h2>Dein Browser ist soooooo Standard</h2>
     </p>
     <p><small>(Wir haben auch ein paar wirklich gute Argumente.)</small></p>
 
-    <a href="{{ url('firefox') }}" class="mzp-c-button mzp-t-product" data-cta-text="Ich schau mir Firefox mal an" data-cta-type="button" data-cta-position="banner">Ich schau mir Firefox mal an</a>
+    <a href="/de/firefox/challenge-the-default/" class="mzp-c-button mzp-t-product" data-cta-text="Ich schau mir Firefox mal an" data-cta-type="button" data-cta-position="banner">Ich schau mir Firefox mal an</a>
```

Comment:
This Is DE promo include with hardcoded de strings, so I felt it's okay to also hardcode a /de/* link, given it's not available in the defined url()s otherwise.

Reply:
BTW the only thing I have left as-is is the switch logic that still implies the original /firefox takeover: https://github.com/mozilla/bedrock/blob/36e8e9c3c6dd2ae9542743a37c7d8792fafaf8c1/bedrock/firefox/templates/firefox/challenge-the-default/landing-switch.html#L20

which results in missing strings if the campaign is turned off, and the landing not removed and/or redirected away (see e.g. https://www.allizom.org/de/firefox/challenge-the-default/ …)

I contemplated monkey-patching a meta-redirect instead of said include, but the page should be removed or redirected from when the time comes instead of flipping the switch anyways now when it's not /firefox takeover, so I kept it as-is — maybe the switch logic can be removed completely to be on the safe side?